articles/quickstart/webapp/rails/01-login.md

```diff
@@ -311,9 +317,9 @@ With those elements in place, Rails will convert the login link to POST the CSRF
 
 ### ActionDispatch::Cookies::CookieOverflow
 
-This error means that a cookie session is being used and because the whole profile is being stored, it overflows the max-size of 4 kb. If you are unable to access the user profile and you get an error similar to `NoMethodError`, `undefined method '[]' for nil:NilClass`, try using In-Memory store for development.
+This error means that a cookie session is being used and because the whole profile is being stored, it overflows the max-size of 4 kb. If you are unable to access the user profile, or you get an error similar to `NoMethodError`, `undefined method '[]' for nil:NilClass`, try using In-Memory store for development.
 
-Go to `/config/initializers/session_store.rb` and add the following:
+Go to `/config/initializers/session_store.rb` (or create it if it does not exist) and add the following:
```

Comment:
This file does not exist when creating new apps using Rails 5.1.1+ ([source](https://stackoverflow.com/q/44662545))

Reply:
Do requirements need updated, also does it work still with 4.x?

`System requirements: Ruby 2.3.1+ | Rails 5.0.0+ or Rails 4.2.0+`

Reply:
Just tested and those system requirements are still fine 👍